kolibri/locale/es_ES/LC_MESSAGES/kolibri.core.default_frontend-messages.json

```
@@ -41,7 +41,7 @@
   "CommonCoreStrings.classesLabel": "Clases",
   "CommonCoreStrings.clearAction": "Eliminar",
   "CommonCoreStrings.closeAction": "Cerrar",
-  "CommonCoreStrings.coachLabel": "Tutorizar",
+  "CommonCoreStrings.coachLabel": "Tutor",
```

```suggestion
  "CommonCoreStrings.coachLabel": "Tutor",
```
I'm confused why is this being changed to `Tutor` again, when I explicitly set to be `Tutorizar` back in December? Same below  for `CommonCoachStrings.coachLabel`

---

Weird regression, my change on Crowdin was probably overwritten by an intermediate upload. 

@indirectlylit I'm going to approve so we can test if the in-context server can be made to work with this , and I'll have to correct these 2 strings again on Crowdin.